pipeline/dwi_masking.py

```diff
@@ -291,22 +291,23 @@ def npy_to_nifti(b0_normalized_cases, cases_mask_arr, sub_name, view='default',
         # Neural Network Predicted Mask
         CNN_predict_file = subject_name[:len(subject_name) - (len(format) + 1)] + '-' + view + '_originalMask.nii.gz'
         CNN_output_file = path.join(output_dir, CNN_predict_file)
-        bashCommand = 'mv ' + filled_file + " " + CNN_output_file
+        bashCommand = 'cp ' + filled_file + " " + CNN_output_file
         process = subprocess.Popen(bashCommand.split(), stdout=subprocess.PIPE)
         output, error = process.communicate()
 
         output_filter_file = subject_name[:len(subject_name) - (len(format) + 1)] + '-' + view + '_FilteredMask.nii.gz'
         output_mask_filtered = path.join(output_dir, output_filter_file)
 
-        print('Cleaning up ', CNN_output_file)
-        # mask_filter = "maskfilter -force " + CNN_output_file + " -scale 2 clean " + output_mask_filtered
-        mask_filter = path.join(path.dirname(__file__),'../src/maskfilter') + f' {CNN_output_file} 2 {output_mask_filtered}'
+        #print('Cleaning up ', CNN_output_file)
+        mask_filter = "maskfilter -force " + CNN_output_file + " -scale 2 clean " + output_mask_filtered
+        #mask_filter = path.join(path.dirname(__file__),'../src/maskfilter') + f' {CNN_output_file} 2 {output_mask_filtered}'
```

Comment:
@SenthilCaesar , did you accidentally disable my maskfilter?

Reply:
@tashrifbillah . Nope, mrtrix maskfilter output was little better compared to the python maskfilter . I discussed this with Suheyla. We might have to meet with her and finalize which one to use.

Reply:
For future changes like this, put a comment so I know what you guys are doing.

Reply:
Moreover, if you haven't finalized anything, let's not make changes.